po/zh_CN.po

```diff
@@ -724,205 +724,223 @@ msgstr ""
 
 #: bmerge.c:102
 msgid "Intrnal error: nqmaxgrpArg is not a positive length-1 integer vector"
-msgstr ""
+msgstr "内部错误：nqmaxgrpArg不是长度为1的正整型向量"
 
 #: bmerge.c:111
 msgid "Internal error in allocating memory for non-equi join"
-msgstr ""
+msgstr "不等值联结分配内存出现内部错误"
 
 #: bmerge.c:156
 msgid "Internal error: xoArg is not an integer vector"
-msgstr ""
+msgstr "内部错误：xoArg不是整型向量"
 
 #: bmerge.c:271 bmerge.c:379
 #, c-format
 msgid ""
 "Internal error in bmerge_r for '%s' column. Unrecognized value op[col]=%d"
 msgstr ""
+"bmerge_r 针对 '%s' 列的操作出现内部错误。无法识别值 op[col]=%d"
 
 #: bmerge.c:303
 #, c-format
 msgid "Only '==' operator is supported for columns of type %s."
-msgstr ""
+msgstr "%s 类型的列仅支持 '==' 操作符。"
 
 #: bmerge.c:410
 #, c-format
 msgid "Type '%s' not supported for joining/merging"
-msgstr ""
+msgstr "'%s' 类型不支持联结/归并"
 
 #: bmerge.c:468
 msgid "Internal error: xlow!=xupp-1 || xlow<xlowIn || xupp>xuppIn"
-msgstr ""
+msgstr "内部错误：xlow!=xupp-1 或 xlow<xlowIn 或 xupp>xuppIn"
 
 #: chmatch.c:4
 #, c-format
 msgid "x is type '%s' (must be 'character' or NULL)"
-msgstr ""
+msgstr "x 类型为 '%s' （必须为'character'或 NULL）"
 
 #: chmatch.c:5
 #, c-format
 msgid "table is type '%s' (must be 'character' or NULL)"
-msgstr ""
+msgstr "table 类型为 '%s' （必须为 'character' 或 NULL）"
 
 #: chmatch.c:6
 msgid "Internal error: either chin or chmatchdup should be true not both"
-msgstr ""
+msgstr "内部错误：chin 和 chmatchdup 不能同时为真"
 
 #: chmatch.c:30
 #, c-format
 msgid ""
 "Internal error: CHARSXP '%s' has a negative truelength (%d). Please file an "
 "issue on the data.table tracker."
 msgstr ""
+"内部错误：CHARSXP '%s' 的 truelength (%d) 为负。"
+"请将此问题汇报给 data.table 问题追踪器。"
 
 #: chmatch.c:60
 #, c-format
 msgid ""
 "Failed to allocate %lld bytes working memory in chmatchdup: length(table)=%d "
 "length(unique(table))=%d"
 msgstr ""
+"无法为 chmatchdup: length(table)=%d length(unique(table))=%d 分配 %lld 字节的内存空间"
```

Comment:
OK I figured this one out too!!

`msgid` is using `%lld->%d->%d` but `msgstr` is using `%d->%d->%lld`.

It seems pretty clear what's intended to me, but apparently `xgettext` is having trouble processing. We have to number the messages.

Also, the numbering will be `1,2,3` (i.e. the numbering occurs across both types, `%d` and `%lld`).